test/TestAll.cpp

```diff
@@ -0,0 +1,13 @@
+#include <gtest/gtest.h>
+#include "CollisionDetectorTest.cpp"
+#include "GameObjectTest.cpp"
+#include "GameStateHandlertest.cpp"
+#include "HitboxTest.cpp"
+#include "KeyboardListenerTest.cpp"
+#include "MissileLauncherTest.cpp"
```

Comment:
You shouldn't ever need to include cpp files, that's a recipe for circular dependencies. Also, gtest is supposed ot have auotomatic test discovery that doesn't need this.